src/utils/table.ts

```diff
@@ -4,17 +4,21 @@ import { DEFAULT_TABLE_COLUMNS, DEFAULT_TABLE_ROWS } from "settings";
 
 import { Table } from "types/Table";
 
+const generateUniqueTableReference = (title: string) => `${title}-${uuid()}`;
+
 export const generateTable: (props: {
   tableNumber: number;
   columns?: number;
   rows?: number;
+  generateTableReference?: (title: string) => string;
 }) => Table = ({
   tableNumber,
   columns = DEFAULT_TABLE_COLUMNS,
   rows = DEFAULT_TABLE_ROWS,
+  generateTableReference = generateUniqueTableReference,
 }) => {
   const title = `Table ${tableNumber}`;
-  const reference = `${title}-${uuid()}`;
+  const reference = generateTableReference(title);
```

Comment:
It still is going to create unique id for every user, or am I missing something?

Reply:
The change in `src/settings/index.ts` passes through a dumb function that makes the reference the same as the table title. **Only** for the default tables.

Reply:
Ah, I see. Yeah, makes sense